.gitignore

```diff
@@ -2,3 +2,5 @@ __pycache__/
 **/.DS_Store
 **/.db
 .python-version
+node_modules
+package-lock.json
```

Comment:
The `package-lock.json` should actually be checked in (see [docs](https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json))!

Reply:
Correct